backvendor/gosource.go

```diff
@@ -30,9 +31,16 @@ import (
 	"github.com/release-engineering/backvendor/backvendor/glide"
 )
 
-type RepoRoot struct {
+var vcsRepoRootForImportPath = vcs.RepoRootForImportPath
+
+type RepoPath struct {
 	vcs.RepoRoot
 
+	// RootPath is the path to the top-level package relative to
+	// the root of the repository, or "" if Path is the root of
+	// the repository.
+	RootPath string
```

Comment:
What do you think about naming this `RelRootPath`? `RootPath` sounds like it's the absolute path to the root of the repository.

Reply:
Also, what is Path in the comment here? Looks like the comment is a copy from another struct.
Should TopLevel be in the name as that seems to be the key thing?

Reply:
How about SubPath? It's meant to be the "remainder" of the import path once the import path corresponding to the root of the repository is known.

E.g. for an import path of "github.com/kubernetes/autoscaler/cluster-autoscaler", there will be a `vcs.RepoRoot{Root: "github.com/kubernetes/autoscaler"}`, and I want to capture "cluster-autoscaler" -- the path under the repository's root.

Reply:
@twaugh, that sounds fine.